JS/Planes/MilitaryPlane.js

```diff
@@ -1,16 +1,20 @@
-const Plane = require('./Plane');
+const Plane = require("./Plane");
 
 class MilitaryPlane extends Plane {
+  constructor(
+    model,
+    maxSpeed,
+    maxFlightDistance,
+    maxLoadCapacity,
+    militaryType
+  ) {
+    super(model, maxSpeed, maxFlightDistance, maxLoadCapacity);
```

Comment:
This constructor has too many arguments. It would be a good idea to pass a single object to a constructor. This object will have properties: model, maxSpeed, etc.